presto-hive/src/main/java/com/facebook/presto/hive/HiveWriterFactory.java

```diff
@@ -102,6 +106,7 @@
 {
     private static final int MAX_BUCKET_COUNT = 100_000;
     private static final int BUCKET_NUMBER_PADDING = Integer.toString(MAX_BUCKET_COUNT - 1).length();
+    private static final Pattern BUCKET_FILE_NAME_PATTERN = Pattern.compile("\\d{8}_\\d{6}_\\d{5}_[a-z0-9]{5}_bucket-(\\d+)(\\..*)?");
```

Comment:
Should the file prefix be included in this pattern? What do you think about making it simply `_bucket-(\\d+)(\\..*)?`. This should be more flexible in case file name prefix ever changes. Can we see any downsides of reducing the matching pattern?

Reply:
Yeah I agree with @arhimondr. Ideally this function should be a reverse function for `computeBucketedFileName`, which means we would leave prefix aside. Would the caller be aware of the prefix?

Reply:
Thank you @shixuan-fan, REGEX was updated